vllm/lora/layers.py

```diff
@@ -64,6 +64,64 @@ def dec(*args, **kwargs):
     return dec
 
 
+def apply_bias(
```

Comment:
Would it be possible to add this inside `PunicaWrapper.add_lora()`? There could be an optional `bias` argument in `add_lora()` and then the logic of testing whether the bias is `None` and doing the index computation could be moved inside this function. It seems to me that it would eliminate repeated code lines in this file, but I don't know all the details.

Reply:
@maxdebayser I don't have a strong opinion, but this was to keep the changes out of punica wrapper because of it not being directly releated to punica.